migrate/createMigrationsTable.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$createMigrationsTable = "CREATE TABLE IF NOT EXISTS `community-voices_migrations` (
+    `id` int(21) NOT NULL AUTO_INCREMENT,
+    `script_name` VARCHAR(250) NOT NULL,
+    `arguments` TEXT DEFAULT NULL,
```

Comment:
Possibly considering dropping this column because I do not think it will get used, but will investigate.